File: C#/Q01e.cs
using System;

namespace Q01e_2020
{
    class Program
    {
        private static int sumOfNumbers(int [] listNumbers)
        {
            int amount = 0;
            foreach (int x in listNumbers)
            {
                if (x > 22 && x <= 40)
                {
                    amount = amount + x;
                }
            }
            return amount;
        }

        static void Main(String[] args)
        {
            int [] numArray = { 23, 22, 41, 26, 55, 27, 34, 45, 24, 40, 25, 29, 51, 28, 55 };

            int total = sumOfNumbers(numArray);
            String message = "Sum of selected numbers is " ;
            Console.WriteLine(message + Convert.ToString(total));

            Console.ReadKey();
        }
    }
}

File: Java/Q01e.java
public class Q01e
{
    private static int sumOfNumbers(int listNumbers[])
		{
            int amount = 0;
            for(int x=0; x<listNumbers.length; x++)
                {
                    if (listNumbers[x] > 22 && listNumbers[x] <= 40)
                        {
                            amount = amount + listNumbers[x];
                        }
                }
            return amount;
		}

    public static void main(String[] args)
        {
            int numArray [] = {23, 22, 41, 26, 55, 27, 34, 45, 24, 40, 25, 29, 51, 28, 55};
	
            String message = "Sum of selected numbers is ";
            int total = sumOfNumbers(numArray);
            System.out.println(message + total);
        }
}